week-1.py

```diff
@@ -50,17 +52,21 @@ def play(self, dealerCard):
         
         if playerCard < dealerCard:
             # [INCREMENT THE PLAYER'S POT, AND RETURN A MESSAGE]
+            self.playerpot -= gameStake
+            return "lost" + str (playerCard)
         else:
             # [INCREMENT THE PLAYER'S POT, AND RETURN A MESSAGE]
-        
+            self.playerpot -= gameStake
```

Comment:
Right now all your players are ending the game with $0 dollars. This is because you are decreasing from their pots no matter if they win or lose. To fix this you should use += instead of -= here to add to the pot when the player wins.